kubernetes/baseline-attack.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
+# shellcheck disable=SC2001
+
+OUTPUT_DIR=$1
+
+/go_attack/kubernetes/log-git-commit.sh "$OUTPUT_DIR"
+
+python3 go_attack/scripts/baseline_attack.py \
+  --config go_attack/configs/gtp-base.cfg \
```

Comment:
Is gtp-raw.cfg ever used? (I don't see it searching through this file)

Reply:
I believe we currently only regularly use it for transfer experiments (albeit with modifications — the current contents of `maxVisits0=32` + `numSearchThreads=6` is usually not what we want). For baseline attack we use usually use KataGo-custom instead of KataGo-raw since we want to support pass-hardening so using gtp-raw.cfg might not be appropriate here